docs/design/autoware-architecture/perception/reference_implementation.md

```diff
@@ -0,0 +1,72 @@
+# Perception component Reference Implementation Design
+
+## Purpose of this document
+
+This document outlines detailed design of the reference imprementations. This allows developers and users to understand what is currently available with the Perception Component, how to utilize, expand, or add to its features.
+
+## Architecture
+
+This diagram describes the architecture of the reference implementation.
+
+![overall-perception-architecture](image/reference-implementaion-perception-diagram.drawio.svg)
+
+The Perception component consists of the following sub-components:
+
+- **Obstacle Segmentation**: Identifies point clouds originating from obstacles(not only dynamic objects but also static obstacles that should be avoided, such as stationary obstacles) that the ego vehicle should avoid. For example, construction cones are recognized using this module.
+- **Occupancy Grid Map**: Detects blind spots (areas where no information is available and where dynamic objects may jump out).
+- **Object Recognition**: Recognizes dynamic objects surrounding the ego vehicle in the current frame and predicts their future trajectories.
+  - **Detection**: Detects the pose and velocity of dynamic objects such as vehicles and pedestrians.
+    - **Detector**: Triggers object detection processing frame by frame.
+    - **Interpolator**: Maintains stable object detection. Even if the output from Detector suddenly becomes unavailable, Interpolator uses the output from the Tracking module to maintain the detection results without missing any objects.
+  - **Tracking**: Associates detected results across multiple frames.
+  - **Prediction**: Predicts trajectories of dynamic objects.
+- **Traffic Light Recognition**: Recognizes the colors of traffic lights and the directions of arrow signals.
+
+The following diagram describes the reference implementation of the Perception component. By adding new modules or extending the functionalities, various ODDs can be supported.
+
+### Details
```

Comment:
Although these detailed explanation are very helpful for users, I believe these details are bit too detailed for long term maintenance, with duplicated description both existing in Autoware Documentation and Autoware README.md. I am a bit concerned that keeping these description up-to-date may take negligible resources.

 How about we just put a link to the readme of each packages instead of writing down the description here?

Reply:
Thanks! I removed details.